### DISCUSSION TITLE
Kentimata under oligon in Stathis font

### DISCUSSION THREAD

**Discussion by Beatl3** [neanes/neanes#1151] [Q&A]
When the kentimata are under the oligon, is there any way to have them to the right part of the oligon instead of having them in the middle, when using the Stathis font?

**Comment by ilizol** [comment]
Stathis font in Neanes has the same issue that I had with Almouzios font. The "oligonKentimataBelow" has an alternative version "oligonKentimataBelow.alt01" needed for the combination with psifiston (ψηφιστόν). The "oligonKentimataBelow" is good to have the kentimata to the right part and the "oligonKentimataBelow.alt01" in the middle to be enough room for the psifiston.
Please check the fix in Almouzios font to help you understand what I mean:
https://github.com/ilizol/Almouzios/commit/be445748c15259da87bafc36e6ebddc60adb0cf7

**Comment by danielgarthur** [comment]
This was an oversight and has been fixed in the latest release.

**Reply by ilizol** [reply]
Before:
![IMG_20241211_064159.jpg](https://github.com/user-attachments/assets/639131bf-8d7d-42f9-ad12-2038d714bfd0)

After:
![IMG_20241211_064317.jpg](https://github.com/user-attachments/assets/5447f2b6-f182-44ef-a4e8-7a0a13bed9f0)
